downstream/modules/platform/proc-controller-image-pull-policy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="proc_controller-image-pull-policy_{context}"]
+
+= controller-image-pull-policy
+
+
+. Under *Image Pull Policy*, click on the radio button to select *Always*, *Never*, or *IfNotPresent*.
+. Define the *Image Pull Secrets*
+.. Click on the angle bracket to show the *Add Image Pull Secret*.
```

Comment:
Value is the only field there is. So that is still accurate. A good question on whether or not there are values, I have sent an inquiry to Christian, though I suspect that the answer is that it depends upon the user's hardware specs.

Reply:
The value is the name of the k8s secret that contains credentials for accessing a private image registry.  The user would know the name of the pull secret because they will have created it.  

![image](https://user-images.githubusercontent.com/11698892/193652814-79fe4c4a-f0d4-4e97-ac4c-5c6ddf10c136.png)


Note that this can be plural (Image Pull Secret (s)).  


Docs on creating image pull secrets:
* https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
* https://docs.openshift.com/container-platform/4.10/openshift_images/managing_images/using-image-pull-secrets.html

Reply:
Also, this is optional.  If customers are only using images from the official redhat image registry [redhat.registry.io](http://redhat.registry.io/) (which is the default), then they do not need to create and specify an image pull secret.